Visual Studio does not like this

(required for static build)

--- c++/etip.h.in
+++ c++/etip.h.in
@@ -341,7 +341,6 @@
 #  else
 #     include <iostream.h>
 #  endif
-   extern "C" void exit(int);
 #endif
 
 inline void THROW(const NCursesException *e) {
